rpc/flipt/ofrep/ofrep.proto

```diff
@@ -29,7 +31,21 @@ message FlagEvaluation {
   repeated string supported_types = 1;
 }
 
+message EvaluateFlagRequest {
+  string key = 1;
+  map<string, string> context = 2;
+}
+
+message EvaluatedFlag {
+  string key = 1;
+  string reason = 2;
```

Comment:
This is a string instead of a _enum_ because of the translation thing on the _grpc gateway_, to move with a enum we're gonna need to ignore the buf linter in order to allow creating an _enum_ with the prefix as the same as the name

Reply:
it might be nice to make this an enum as you said. im ok with telling buf to ignore the lint errors. wdyt?